auth_jwt/models/auth_jwt_validator.py

```diff
@@ -97,6 +97,12 @@ class AuthJwtValidator(models.Model):
     cookie_secure = fields.Boolean(
         default=True, help="Set to false only for development without https."
     )
+    renew_cookie_on_response = fields.Boolean(
+        help="Renew the cookie in every response to stay the client "
+        "authenticatedas long it use the API. Don't mark unless you have a "
+        "way to invalidate sessions",
+        default=False,
```

Comment:
```suggestion
        default=True,
```

for backward compatibility